### DISCUSSION TITLE
Customize menu options

### DISCUSSION THREAD

**Discussion by pointy91** [rxhanson/Rectangle#681] [Q&A]
Hello,
 is there a way to remove options from menu dopdown?
  i'd like to have a few i se and hide the rest, which clutter the menu.
Also is there a way to keep a window ot top, 
or an action to gather / send all windows to specific display?
Thanks

**Comment by rxhanson** [comment]
> is there a way to remove options from menu dopdown?

Currently no, but I'm not opposed to it. My bandwidth is pretty limited, but I'd be up for merging a good pull request for it.

> Also is there a way to keep a window ot top,

No. This is a macOS limitation, but check out Todo Mode in the preferences. 

> or an action to gather / send all windows to specific display?

Multiple window controls are in my paid superset of Rectangle, Hookshot (https://hookshot.app), and I don't have plans to add them into Rectangle in the interest of keeping Rectangle simple.
